src/CollectionCardMaker.php

```diff
@@ -0,0 +1,26 @@
+<?php
+
+class CollectionCardMaker {
```

Comment:
this is a pretty good class name because it describes the purpose of the class pretty accurately, however I would say that this class literally represents a game, because it has the same properties as a game. so i would call it Game or BoardGame, whichever is consistently used throughout the code and db